### PR TITLE
[서민아] 프로토타입 설계

### PR DESCRIPTION
**Pull request by esthel7**
Co-author-by: 5nxtnxtnxt <5nxtnxtnxt@gmail.com>
Co-author-by: ChanWoo Kim <cwkim0321@gmail.com>
Co-author-by: yunuo46 <yunuo468@gmail.com>
Co-author-by: Jinsoo Choi <jinsoo2005@gmail.com>

### CODE REVIEW

**Comment by peageon** [comment]
LGTM

**Review by 5nxtnxtnxt — APPROVED** [review]
LGTM🙇‍♂️

File: front/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "trailingComma": "none",
+  "arrowParens": "avoid",
+  "endOfLine": "auto"
+}
```
Comment (kcwww) [inline front/.prettierrc:10]: 좋네요 LGTM

File: front/src/App.css [hunk 2]
```diff
@@ -1,38 +1,10 @@
-.App {
-  text-align: center;
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #14212B;
 }
```
Comment (kcwww) [inline front/src/App.css:5]: 나중에 변수명으로 해요

**Review by kcwww — APPROVED** [review]
LGTM

**Pull request merged**